cypress/support/index.js

```diff
@@ -1,6 +1,22 @@
 Cypress.Commands.add('login', () => {
+  cy.server();
+  cy.route('/user-config/config.json').as('getUserConfig');
```

Comment:
we can also supply a custom config using fixtures instead of generating one in the CI process

Reply:
That's a good point. Having a fixture for the user config will allow us to run tests even locally without messing with the config file we might have in the working directory.